WordPress/gutenberg
# Link Control test migration: Should contain a label when it should open in a new tab

## What?


Part of https://github.com/WordPress/gutenberg/issues/50995
Based on top of https://github.com/WordPress/gutenberg/pull/50996
This Pr needs to wait for the previous one to get merged since we didn't have any playwright tests for this component yet, it needed that setup.
This PR migrates the test "Should contain a label when it should open in a new tab" for the link control component to playwright

## Why?

In an effort to migrate our e2e tests to use playwright, we've decided to pick up some of the tests for this component as we work on it to improve it, instead of trying to maintain the puppeteer tests.

## How?

Following the [guidelines](https://github.com/WordPress/gutenberg/blob/971ed6c8506140421b97e38a1df77ad2666c67da/docs/contributors/code/e2e/migration.md) for e2e tests migrations

## Testing Instructions




run `npm run test:e2e:playwright -- -g "Should contain a label when it should open in a new tab"` and the test should pass
